scripts/ci/install_docker.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# ARM64 image does not have docker, install manually
+
+if [[ $(dpkg --print-architecture) == "amd64" ]]; then
+    echo "Docker is already installed on AMD64"
+    exit 0
+fi
+# https://docs.docker.com/engine/security/rootless/
+/bin/bash -c "$(curl -fsSL https://get.docker.com)"
```

Comment:
I strongly feel we should persuade 1ES to provide an image with docker, instead of installing docker by ourselves.

Reply:
Hope one day this issue will be fixed. https://github.com/actions/runner-images/issues/5631